installer/x86_64/platforms/x86_64-accton_as7512_32x-r0

```diff
@@ -0,0 +1,3 @@
+ONSOLE_PORT=0x2f8
+CONSOLE_DEV=1
+CONSOLE_SPEED=115200
```

Comment:
double checked - it stuck with 9600 speed. I assume it is not correct to connect to console on other speed than it`s work.